src/config.ts

```diff
@@ -30,14 +30,18 @@ function isConfig(c: ConfigFile): c is Config {
   return (c as Config).name?.length > 0 && typeof (c as Config).monorepo === 'object';
 }
 
-async function getConfigFiles(dir: string): Promise<Config[]> {
+/**
+ * Reads all pipeline.foo.yml files from .buildkite/ relative to a given basePath, and returns file locations
+ */
+async function getConfigFiles(basePath: string): Promise<ConfigFile[]> {
   return fs
-    .readdir(dir)
+    .readdir(path.join(basePath, CONFIG_REL_PATH))
```

Comment:
Reviewed how we load configs, and there's nothing to stop us recursing through all the directories to look for valid pipeline files if we want to support that :+1: